config/Dockerfiles/tests.d/libvirt/06_libvirt-multi-node-network

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -xe
+
+# Verify basic provisioning for all supplied providers
+# distros.exclude: centos7 fedora30
+# providers.include: libvirt
+
+## NOTE: This is a temporary test to run libvirt tests until we fix
+## the networking race condition. Tests will only run on centos7.
+
+set -o pipefail
+
+DISTRO=${1}
+PROVIDER=${2}
+
+TEST_NAME="${DISTRO}/${PROVIDER}/06_libvirt-multi-node-network"
+DESCRIPTION="Test ${PROVIDER} provider with multi-node-network provisioning"
+
+echo "Test Name: ${TEST_NAME}"
+echo "${DESCRIPTION}"
+
+TARGET="libvirt-multi-node-net"
+
+pushd docs/source/examples/workspaces/${PROVIDER}
+
+function clean_up {
+    set +e
+    linchpin -w . -v destroy ${TARGET}
+    # remove output file used to gen uhash if it exists
+    rm resources/ -rf || true
+}
+trap clean_up EXIT SIGHUP SIGINT SIGTERM
+
+linchpin -w . -v up ${TARGET} 
+pushd resources/
+IP_NUM=$(cat ${TARGET}.output|grep -o ip|wc -l)
+if [ ${IP_NUM} -eq '6']; then
```

Comment:
The test check if multinet, multinode works - good
But you need also to verify if reserved IP works.

Reply:
If you want to really test the static ip, you need to wait long time to see if it's changing. But basically as far as the task-(update ip in inventory) runs without any error, the ips will be permanently changed (This is what customer said and I tested it, it worked). So, if this test can pass, the static-ip will work. In another word, this test also test the static-ip

Reply:
How about query libvirt \ DHCP to show that the IP is reserved?